.github/workflows/build-and-test.yml

```diff
@@ -23,3 +26,11 @@ jobs:
         source .profile.d/chrome-for-testing.sh
         chrome --version
         chromedriver --version
+  done:
+    runs-on: ubuntu-latest
+
+    needs: [test]
+
+    steps:
+      - run: echo done
+        working-directory: /
```

Comment:
This isn't needed unless a repo uses changeling + optional jobs and has to fake the final required job etc.

```suggestion
```